arm-templates/ExampleWithPgBouncer/README.md

```diff
@@ -0,0 +1,49 @@
+# Configuring Connection Pooling using PgBouncer
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-postgresql%2Fmaster%2Farm-templates%2FExampleWithPgBouncer%2Ftemplate.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png" />
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-postgresql%2Fmaster%2Farm-templates%2FExampleWithPgBouncer%2Ftemplate.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+PgBouncer is a lightweight connection pooler that can be installed on the virtual machine (VM) running the application. The application connects to the PgBouncer proxy service running locally on the VM while PgBouncer service in-turn connects to the Azure Database for PostgreSQL service using the credentials and configuration settings specified in the pgbouncer.ini file. The maximum number of connections and default pool size can be defined in the configuration settings in pgbouncer.ini.
```

Comment:
provide a link to pgbouncer's homepage

Reply:
Give a very brief overview of what this template achieves

Reply:
For example the intro can look like this (rough draft, more to show the sections):

Pgbouncer [link] is a lightweight connection pooler for Postgres. Connection pooling can improve your performance[link to blog] by reducing connection churn which is a costly operation in Postgres.

Pgbouncer can be installed on the virtual... in pgbouncer,ini.

This ARM template creates (...?), sets up (...?) and finally (...?). 

@savjani